backend/controllers/history.controller.js

```diff
@@ -0,0 +1,82 @@
+import historyVideos from '../models/history.model.js'
+
+//@route   GET api/history
+//@desc    Get All history Videos
+//@access  Private
+const getAllHistoryVideos = async (req, res) => {
+    try {
+        const id = req.user;
+        const videos = await historyVideos.findOne({ user: id })
+        res.status(200).send(videos)
+    } catch (err) {
+        res.status(500).json({ success: false, message: 'Server Error' })
+    }
+}
+
+//@route   POST api/history
+//@desc    Add Video as LikedVideo
+//@access  Private
+
+const addVideoToHistory = async (req, res) => {
+    try {
+        const id = req.user
+        const videos = await historyVideos.findOne({ user: id })
+        const { videoID } = req.body;
+        const isPresent = videos.historyvideos.some((v) => v === videoID)
```

Comment:
Change the naming to `historyVideos`

Reply:
historyVideos is name of my model and historyvideos is the field name in that model.
So here it couldn't be changed :)